src/Pos/Ssc/GodTossing/DB.hs

```diff
@@ -0,0 +1,58 @@
+-- | DB operations for storing and dumping GtGlobalState.
+
+module Pos.Ssc.GodTossing.DB
+       ( getGtGlobalState
+       , gtGlobalStateToBatch
+       , prepareGtDB
+       ) where
+
+import           Data.Default                   (def)
+import qualified Data.Text.Buildable
+import qualified Database.RocksDB               as Rocks
+import           Universum
+
+import           Formatting                     (bprint, build, (%))
+import           Pos.Binary.Class               (encodeStrict)
+import           Pos.Binary.Ssc                 ()
+import           Pos.DB                         (MonadDB, RocksBatchOp (..))
+import           Pos.DB.Error                   (DBError (DBMalformed))
+import           Pos.DB.GState.Common           (gsGetBi, gsPutBi)
+import           Pos.Ssc.GodTossing.Core        (VssCertificatesMap)
+import           Pos.Ssc.GodTossing.Types       (GtGlobalState (..))
+import qualified Pos.Ssc.GodTossing.VssCertData as VCD
+import           Pos.Util.Util                  (maybeThrow)
+
+getGtGlobalState :: MonadDB m => m GtGlobalState
+getGtGlobalState =
+    maybeThrow (DBMalformed "GodTossing global state DB is not initialized") =<<
+    gsGetBi gtKey
+
+gtGlobalStateToBatch :: GtGlobalState -> GtOp
+gtGlobalStateToBatch = PutGlobalState
+
+prepareGtDB :: MonadDB m => VssCertificatesMap -> m ()
+prepareGtDB certs =
+    whenNothingM_ (gsGetBi @_ @GtGlobalState gtKey) $
+        gsPutBi gtKey (def {_gsVssCertificates = vcd})
+  where
+    vcd = foldr' VCD.insert VCD.empty . toList $ certs
```

Comment:
It'd be nicer if it was a helper function in `Pos.Ssc.GodTossing.VssCertData` (e.g. `fromList`)